lib/active_model/serializer/adapter/json.rb

```diff
@@ -7,37 +7,13 @@ class Json < Adapter
         def serializable_hash(options = nil)
           options ||= {}
           if serializer.respond_to?(:each)
-            @result = serializer.map { |s| FlattenJson.new(s).serializable_hash(options) }
```

Comment:
I didn't like the way multiple methods were modifying the same instance variable.  :-/